joomla/joomla-cms
# [com_checkin] checkin view: Add missing tooltip in search

#### Summary of Changes

Simple PR to add missing tootip to com_checkin search filter.
#### Testing Instructions

Code review.

or
1. Apply patch
2. Go to "System" -> "Check-in", check the tootip in the search field.
